adhocracy4/images/assets/image_uploader.js

```diff
@@ -1,27 +1,29 @@
-/* global $ */
 function init () {
-  const clearInputs = $('input[data-upload-clear]')
-  const previewImages = $('img[data-upload-preview]')
+  const clearInputs = document.querySelectorAll('input[data-upload-clear]')
+  const previewImages = document.querySelectorAll('img[data-upload-preview]')
 
-  previewImages.each(function (index, previewImage) {
-    previewImage = $(previewImage)
-    const inputId = previewImage.data('uploadPreview')
-    const clearInput = clearInputs.filter('[data-upload-clear="' + inputId + '"]')
-    $('#' + inputId).change(function (e) {
+  previewImages.forEach(function (previewImage, index) {
+    previewImage = document.getElementById(previewImage.id)
+    const inputId = previewImage.dataset.uploadPreview
+    const clearInput = Array.from(clearInputs).filter(el => el.matches('[data-upload-clear="' + inputId + '"]'))
+    document.querySelector('#' + inputId).addEventListener('change', function (e) {
       const domInput = e.target
       if (domInput.files && domInput.files[0]) {
         const name = domInput.files[0].name
-        $('#text-' + inputId).val(name)
+        const text = document.querySelector('#text-' + inputId)
+        if (text != null) {
+          text.value = name
+        }
         previewImage.alt = name
         if (window.FileReader) {
           const reader = new window.FileReader()
-          reader.onload = function (e) {
-            previewImage.attr('src', e.target.result)
+          reader.addEventListener('load', function (e) {
+            previewImage.setAttribute('src', e.target.result)
             clearInput.prop('checked', false)
```

Comment:
in line 26 you rewrote a similar assignment, maybe you want to do it here as well?

Reply:
oh, good catch `prop` is still jquery :facepalm: